### PR TITLE
178289222 -- Gemfile audit

### PR DESCRIPTION
**Pull request by knowuh**
https://www.pivotaltracker.com/story/show/178289222

### CODE REVIEW

**Comment by knowuh** [comment]
Green build and lot fewer Gems installed.

File: docs/core-extensions.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Extensions to Existing Ruby Classes
```
Comment (dougmartin) [inline docs/core-extensions.md:1]: @knowuh did you mean to add this file?  It was added in the commit where you removed `RedCloth`

**Comment by dougmartin** [comment]
@scytacki I've added you to the review list so you can see what gems are going to be removed.

**Review by scytacki — APPROVED** [review]
Looks good to me

File: docs/core-extensions.md (hunk 1 above)
Comment (knowuh) [reply docs/core-extensions.md:1]: Yeah -- I converted a textile document to markdown and committed it. I didn't look at the file too closely, it's probably outdated, and could probably be removed.

**Pull request merged**